README.adoc

```diff
@@ -17,12 +17,12 @@ $ cargo +nightly doc
 
 == Contributing
 
-You can **build and run the whole test suite** with `cargo test`.  The test
-suite runs cleanly and should remain clean.
+You can **build and run the whole test suite** with https://nexte.st/[nextest]:
+`cargo nextest run`.  The test suite runs cleanly and should remain clean.
```

Comment:
Is this necessary? I'm reluctant to add a dependency on a required install unless there's some overwhelming benefit.

Reply:
It doesn't seem to be hugely impactful for test times, but it does make the CI a lot nicer to look at I think. My worry is that at some point cargo test might break — though it's not hugely likely to happen with dropshot because it has smaller/simpler tests than something like omicron.

ymmv, happy to change the readme back, not use it in CI, or both.

Reply:
ok, changed this back